### ISSUE TITLE
platform inconsistent results on spaces

### ISSUE THREAD

**Issue by chadbr** [palantir/tslint#3274]
For some reason - within the last 2 weeks or so... we get unpredictable results for indentation.

-All of our Linux builds give these results
-One of our macs gives these results
-2 of our macs do not give these results
-none of our windows machines give these results

2017-10-03T22:03:30.5959640Z ERROR: /opt/vsts/work/1/s/src/app/input-controls/control-config.ts[16, 1]: Expected indentation of 4 spaces but found 2.
2017-10-03T22:03:30.5998590Z ERROR: /opt/vsts/work/1/s/src/app/input-controls/control-config.ts[17, 1]: Expected indentation of 4 spaces but found 2.
2017-10-03T22:03:30.6025890Z ERROR: /opt/vsts/work/1/s/src/app/input-controls/control-config.ts[20, 1]: Expected indentation of 4 spaces but found 2.

### Bug Report

- node 8.6 -- I thought it could be the node version but it doesn't seem to matter (tried a few on different machines...)

- __TSLint version__: I've tried 5.7, 5.6, 5.5
- __TypeScript version__: 2.5.2 / 2.5.3
- __Running TSLint via__: (pick one) CLI
yarn -- 
    "lint": "ng lint --type-check",

#### TypeScript code being linted

```ts
  decimalPlace?: number;
```

with `tslint.json` configuration:

```json
    "indent": [
      true,
      "spaces",
      2
    ],
```
it used to be this when the messages started popping up
```
    "indent": [
      true,
      "spaces"
    ],
```

#### Actual behavior
see messages above

#### Expected behavior
I don't expect these messages...

-----

any ideas?

**Comment by chadbr** [comment]
complete file:

```
import { GridOptions } from 'ag-grid';

export interface GridConfig {
  tableList: any;
  gridOptions: GridOptions;
  gridName: any;
}
```
```
ERROR: /opt/vsts/work/1/s/src/app/utilities/grid-config.ts[4, 1]: Expected indentation of 4 spaces but found 2.
ERROR: /opt/vsts/work/1/s/src/app/utilities/grid-config.ts[5, 1]: Expected indentation of 4 spaces but found 2.
ERROR: /opt/vsts/work/1/s/src/app/utilities/grid-config.ts[6, 1]: Expected indentation of 4 spaces but found 2.

```

tslint.json
```
{
  "rulesDirectory": [
    "node_modules/codelyzer"
  ],
  "extends": "tslint-config-standard",
  "rules": {
    "space-before-function-paren": [false, "always"],

    "callable-types": true,
    "class-name": true,
    "comment-format": [
      true,
      "check-space"
    ],
    "curly": true,
    "eofline": true,
    "forin": true,
    "import-blacklist": [true, "rxjs"],
    "import-spacing": true,
    "indent": [
      true,
      "spaces",
      2
    ],
    "interface-over-type-literal": true,
    "label-position": true,
    "max-line-length": [
      false
    ],
    "member-access": false,
    "member-ordering": [
      false,
      "static-before-instance",
      "variables-before-functions"
    ],
    "no-arg": true,
    "no-bitwise": true,
    "no-console": [
      true,
      "debug",
      "info",
      "time",
      "timeEnd",
      "trace"
    ],
    "no-construct": true,
    "no-debugger": true,
    "no-duplicate-variable": true,
    "no-empty": false,
    "no-empty-interface": true,
    "no-eval": true,
    "no-inferrable-types": [true, "ignore-params"],
    "no-shadowed-variable": true,
    "no-string-literal": false,
    "no-string-throw": true,
    "no-switch-case-fall-through": true,
    "no-trailing-whitespace": true,
    "no-unused-expression": true,
    "no-use-before-declare": true,
    "no-var-keyword": true,
    "object-literal-sort-keys": false,
    "one-line": [
      true,
      "check-open-brace",
      "check-catch",
      "check-else",
      "check-whitespace"
    ],
    "prefer-const": true,
    "quotemark": [
      true,
      "single"
    ],
    "radix": true,
    "semicolon": [
      "always"
    ],
    "triple-equals": [
      true,
      "allow-null-check"
    ],
    "typedef-whitespace": [
      true,
      {
        "call-signature": "nospace",
        "index-signature": "nospace",
        "parameter": "nospace",
        "property-declaration": "nospace",
        "variable-declaration": "nospace"
      }
    ],
    "typeof-compare": true,
    "unified-signatures": true,
    "variable-name": false,
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator",
      "check-type"
    ],

    "use-input-property-decorator": true,
    "use-output-property-decorator": true,
    "use-host-property-decorator": true,
    "no-input-rename": true,
    "no-output-rename": true,
    "use-life-cycle-interface": true,
    "use-pipe-transform-interface": true,
    "component-class-suffix": true,
    "directive-class-suffix": true,
    "no-access-missing-member": true,
    "templates-use-public": true,
    "invoke-injectable": true,
    "cyclomatic-complexity": [true, 20]
  }
}


from tslint-config-standard npm package...

```
module.exports = {
  extends: ['tslint-eslint-rules'],
  rules: {
    // TSLint rules.
    'member-access': false,
    'member-ordering': [
      true,
      {
        order: [
          'public-static-field',
          'protected-static-field',
          'private-static-field',
          'public-instance-field',
          'protected-instance-field',
          'private-instance-field',
          'constructor',
          'public-static-method',
          'protected-static-method',
          'private-static-method',
          'public-instance-method',
          'protected-instance-method',
          'private-instance-method'
        ]
      }
    ],
    'no-internal-module': true,
    'typedef-whitespace': [
      true,
      {
        'call-signature': 'nospace',
        'index-signature': 'nospace',
        'parameter': 'nospace',
        'property-declaration': 'nospace',
        'variable-declaration': 'nospace'
      },
      {
        'call-signature': 'onespace',
        'index-signature': 'onespace',
        'parameter': 'onespace',
        'property-declaration': 'onespace',
        'variable-declaration': 'onespace'
      }
    ],
    'await-promise': true,
    'curly': [true, 'ignore-same-line'],
    'label-position': true,
    'no-arg': true,
    'no-conditional-assignment': true,
    'no-duplicate-variable': true,
    'no-empty': true,
    'no-eval': true,
    'no-reference-import': true,
    'no-switch-case-fall-through': true,
    'no-unused-expression': [true, 'allow-fast-null-checks', 'allow-tagged-template'],
    'no-unused-variable': [true, { 'ignore-pattern': '^_' }],
    'no-use-before-declare': true,
    'no-var-keyword': true,
    'radix': true,
    'return-undefined': true,
    'triple-equals': [
      true,
      'allow-null-check'
    ],
    'use-isnan': true,
    'eofline': true,
    'ter-indent': [
      true,
      2,
      {
        SwitchCase: 1
      }
    ],
    'no-trailing-whitespace': true,
    'trailing-comma': [
      true,
      {
        'multiline': 'never',
        'singleline': 'never'
      }
    ],
    'class-name': true,
    'comment-format': [
      true,
      'check-space'
    ],
    'jsdoc-format': true,
    'new-parens': true,
    'no-angle-bracket-type-assertion': true,
    'no-consecutive-blank-lines': true,
    'no-floating-promises': true,
    'no-misused-new': true,
    'no-string-throw': true,
    'no-unnecessary-qualifier': true,
    'one-line': [
      true,
      'check-catch',
      'check-finally',
      'check-else',
      'check-open-brace',
      'check-whitespace'
    ],
    'one-variable-per-declaration': true,
    'quotemark': [
      true,
      'single',
      'avoid-escape',
      'jsx-single'
    ],
    'semicolon': [
      true,
      'never'
    ],
    'strict-type-predicates': true,
    'space-before-function-paren': [true, 'always'],
    'unified-signatures': true,
    'variable-name': [
      true,
      'ban-keywords',
      'check-format',
      'allow-leading-underscore',
      'allow-pascal-case'
    ],
    'whitespace': [
      true,
      'check-branch',
      'check-decl',
      'check-operator',
      'check-module',
      'check-type',
      'check-typecast',
      'check-preblock'
    ],
    // TSLint ESLint rules.
    'no-constant-condition': [
      true,
      {
        checkLoops: false
      }
    ],
    'no-control-regex': true,
    'no-duplicate-case': true,
    'no-empty-character-class': true,
    'no-ex-assign': true,
    'no-extra-boolean-cast': true,
    'no-inner-declarations': [
      true,
      'functions'
    ],
    'no-invalid-regexp': true,
    'no-irregular-whitespace': true,
    'no-regex-spaces': true,
    'no-sparse-arrays': true,
    'no-unexpected-multiline': true,
    'valid-typeof': true,
    'ter-arrow-spacing': [
      true,
      {
        'before': true,
        'after': true
      }
    ],
    'no-multi-spaces': true,
    'handle-callback-err': [
      true,
      '^(err|error)$'
    ],
    'block-spacing': [
      true,
      'always'
    ],
    'brace-style': [
      true,
      '1tbs',
      {
        'allowSingleLine': true
      }
    ]
  }
}

```

**Comment by chadbr** [comment]
for now I removed the "tslint-config-standard"...
